tools/depends/README.md

```diff
@@ -53,6 +53,9 @@ Paths below are examples. If you want to build Kodi, follow our **[build guides]
 **webos (buildroot-nc4)**
 `./configure --with-toolchain=/opt/toolchains/arm-webos-linux-gnueabi_sdk-buildroot --prefix=/opt/xbmc-deps --host=arm-webos-linux-gnueabi`
 
+**Steam Deck**
+`./configure --prefix="$HOME/kodi-deps" --host=x86_64-steamdeck-linux-gnu`
+
```

Comment:
I'm not really sold on why this is even needed. Wouldn't you just set --with-rendersystem=gl?

Reply:
Nor am I, but then how to detect the need to disable `CLIENT_PLUGIN_AUTH_GSSAPI_CLIENT`? And what if we need other steamdeck-specific changes? This was helpful because I used to have even more host-dependent changes, but solved them all generically except for `CLIENT_PLUGIN_AUTH_GSSAPI_CLIENT`.

Reply:
I'd rather not prematurely add something until there's really a good reason for it. The changes in this PR aren't it imo

Reply:
Fair. I'll separate the first commit into a new PR and we can revisit this one someday.